pkg/provision/buildroot.go

```diff
@@ -117,13 +106,13 @@ LimitCORE=infinity
 # Uncomment TasksMax if your systemd version supports it.
 # Only systemd 226 and above support this version.
 TasksMax=infinity
-TimeoutStartSec=0
 
 # set delegate yes so that systemd does not reset the cgroups of docker containers
 Delegate=yes
 
 # kill only the docker process, not all processes in the cgroup
 KillMode=process
+OOMScoreAdjust=-500
```

Comment:
how come we dont have this in ubuntu? btw I am curious is this in the standard one ?
it does seem helpful